### PR TITLE
Added a notice to all pages about impending data format expiry

### PR DESCRIPTION
**Pull request by jonheslop**
My first go at HAML, wow. Anyway it all looks ok. I think.

xoxo

(Do not merge until the notice page is made)

### CODE REVIEW

**Comment by gidsg** [comment]
@jonheslop I already made a stub for the page, you can cherry-pick this commit if you like: https://github.com/openregister/registers-frontend/commit/5b589e4b2963c93f49375c3d7eff9f8e45d31e62

**Comment by jonheslop** [comment]
@gidsg oh nice! Cherry picked!

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -34,6 +35,11 @@
   %meta{ name: 'description', content: "#{@page_description.present? ? @page_description : 'GOV.UK Registers provides a secure API that is easy to integrate with so you can get the most up-to-date data from across government.'}" }
   = render partial: 'layouts/analytics'
 
+- content_for :after_header do
+  %aside{class: "notice"}
+    .container
+      %p{data: {"click-events" => "", "click-category" => "Content", "click-action" => "Notice link clicked"}} GOV.UK Registers will stop supporting data formats other than JSON or CSV from 30 September 2018. #{link_to "Read more about format changes", "/data-format-changes"}
```
Comment (gidsg) [inline app/views/layouts/application.html.haml]: we can resolve the path using Rails `#{link_to "Read more about format changes", data_format_changes_path}`.

Comment (jonheslop) [reply app/views/layouts/application.html.haml]: Ah nice, 👌

**Comment by gidsg** [comment]
@jonheslop at wide viewports there is a white band above the grey area, can you take a look?
screen shot 2018-07-05 at 16 12 01

File: app/assets/stylesheets/modules/_notice.scss [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
   p {
     @include core-16;
-    margin: $gutter / 2 0;
+
+    margin: 0;
+    padding: ($gutter / 2) 0;
```
Comment (jonheslop) [inline app/assets/stylesheets/modules/_notice.scss]: 👌

**Comment by injms** [comment]
@gidsg, @mpwoods - can you re-review please?

**Pull request merged**